src/string.cr

```diff
@@ -4318,10 +4498,16 @@ class String
     pointerof(@c)
   end
 
+  # Returns *count* of underlying bytes of this String starting at given *byte_offset* in an **unsafe** way.
```

Comment:
"unsafe way" needs to be more specific otherwise it doesn't mean anything.

Reply:
I copy this from current docs for to_slice method: https://crystal-lang.org/api/0.31.1/String.html#to_slice:Bytes-instance-method . But i think we can remove this comment, because slice involve bound checks?